File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/TemplateDB/Template/TemplateColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace CorrespondenceSystem.TemplateDB.Columns;

[ColumnsScript("TemplateDB.Template")]
[BasedOnRow(typeof(TemplateRow), CheckNames = true)]
public class TemplateColumns
{
    [EditLink]
    public string Title { get; set; }   
    public string TemplateFile { get; set; }

}

File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/ServerTypes/TemplateDB/TemplateColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { TemplateRow } from "./TemplateRow";

export interface TemplateColumns {
    Title: Column<TemplateRow>;
    TemplateFile: Column<TemplateRow>;
}

export class TemplateColumns extends ColumnsBase<TemplateRow> {
    static readonly columnsKey = 'TemplateDB.Template';
    static readonly Fields = fieldsProxy<TemplateColumns>();
}